source/includes_main/_enrollments.md

```diff
@@ -0,0 +1,68 @@
+# Enrollments
+## The enrollment object
+To add a desired prospect to a specific automation you have to create an **enrollment**. The enrollment is then used to describe the current status of the automation for the prospect.
+
+```
+# EXAMPLE OBJECT
+```
+```json
+{
+  "data": {
+    "id": "1",
+    "type": "automations-enrollments",
+    "attributes": {
+      "automation_id": 1,
+      "prospect_id": 1,
+      "current_stay_id": 1,
+      "disenrolled_at": null,
+      "created_at": "2020-01-31T12:00:00.000Z",
+      "updated_at": "2020-01-31T12:00:00.000Z"
+    }
+  }
+}
+```
+
+### Object attributes
+Attribute | Description
+--------- | -----------
+id | **integer** <br />A unique identifier for the enrollment
+automation_id | **integer** <br />The automation's ID this enrollment is associated to
+prospect_id | **integer** <br />The prospect's ID this enrollment is associated to
+current_stay_id | **integer** <br />Define the last stay
```

Comment:
Should we serialize this information? It seems the stay is not used anywhere else in the public api, so it feels strange to have this information here. Actually shouldn't we have instead a current_step_id, which would be the id of the node associated to the current stay?

Reply:
That is my feeling too, I was not sure to keep 'current_stay_id' and add a dedicated section for the Stays? or do what you suggested with the 'current_step_id' 

What others think? @tempo22 @aclarembeau @NelsonEU 

What is the best? @Guaker13

Reply:
The `current_step_id` is the way to go ;) The stays are an internal concept, users don't need it.